inc/auth.php

```diff
@@ -450,15 +450,16 @@ function auth_logoff($keepbc = false) {
  *
  * The info is available through $INFO['ismanager'], too
  *
- * @author Andreas Gohr <andi@splitbrain.org>
+ * @param string $user Username
+ * @param array $groups List of groups the user is in
+ * @param bool $adminonly when true checks if user is admin
+ * @param bool $recache set to true to skip cached results
```

Comment:
Why is the parameter called ‘recache’? Is it redoing cache? $usecache, $enablecache, etc feels more logic for me.

Reply:
Well. I guess the parameter name is right, the description is a bit misleading. Setting it to `true` will repopulate the cache with fresh data. So calling it with `true` is like having no caching, but the result is still stored to be returned on the next `false` call...

Reply:
Ah, of course. Could improve the description please?